conduct/COC_POLICY.md

```diff
@@ -0,0 +1,120 @@
+
+The identity of the harmed individual(s) and the and reporter(s) of an incident report must be kept confidential.
+
+## Conflict of Interest
+
+Any person involved with the management of an incident report must immediately notify the CoC Team in writing and recuse themselves from the process if they have been involved in the incident itself or have any other conflict of interest with any of the involved parties (e.g. same employer).
+
+## Enforcement
+
+### Code of Conduct Team (CoC Team)
+
+The Cross Project Council (CPC) delegates enforcement of the code of conduct to the Code of Conduct Team (CoC Team) through the [CoC Team Charter][].
+
+The structure of the CoC Team and how its members are elected are described in the [CoC Team Charter][].
+
+The current composition of the CoC Team is available in the [code of conduct][CoC] itself along with means to contact by email the whole CoC Team and each of its members individually.
+
+The CoC Team is responsible for maintaining and applying the [incident management processes][].
+
+The CoC Team is empowered to involve external people to resolve an issue provided these commit to abide by the code of conduct and by this policy.
+
+### Delegation to projects
+
+Projects may opt-in to enforce the code of conduct themselves provided they meet the following requirements:
+
+1. Commit to uphold the code of conduct.
+2. Commit to uphold this policy.
+3. Provide a general email address that incidents can be reported to and document it alongside the code of conduct.
+4. Define and publicly document a process by which the project selects people responsible for managing incident reports. For Impact and At Large projects, at least two people must be selected.
+5. List the people responsible for managing incident reports alongside the code of conduct as outlined in the [Reporting incidents section](#reporting-incidents).
+6. Publicly document their decision-making process for responding to incident reports.
+7. Document the foundation's escalation and appeal process alongside the code of conduct.
+8. Participate in good faith in the [escalation](#escalation) and [appeal](#appeals) processes.
+
+## Moderation
+
+Each project is free to implement moderation processes appropriate for the size and scope of the project provided these processes abide by the code of conduct and by this policy.
+
+## Escalation
+
+Reporters, harmed individuals, and accused individuals may [escalate](#escalation) an issue to the CoC Team at any point in the process in case of a conflict of interest, if the report is not being handled appropriately, or in case of an emergency.
+
+The CoC Team is empowered to manage escalation requests of incident reports managed by projects that have opted-in to enforce the code of conduct themselves.
+
+If the CoC Team finds that the escalation request has merit, the CoC Team may decide to takeover the management of the incident report or request changes to the existing management process. 
```

Comment:
I think it's important for projects to be accountable to properly implementing the policies agreed to by project representatives in the CPC.

Reply:
I think that is orthogonal to my point, and is an extremely blunt way to manage this situation that removes nuance that will _always_ exist when something like this comes up. If this ever happens, that as the answer will not suffice.

Reply:
Copying this comment here (breaking out individual comments as requested by @tobie)

Since there isn't consensus on the item above (or, how to reach consensus) is it possible to leave this as is, but then make this a specific part of the follow up activity needed to review CoC updates with each individual project (allowing us to land the PR but expecting we will need to open additional issues specific to the escalation and appeal policy?)

An option here is to open an issue with each project to specifically and openly request approval and also openly document which projects opt-in to managing their own CoC reporting.

Reply:
> An option here is to open an issue with each project to specifically and openly request approval

The CPC _is_ the forum for working this approval out.

> and also openly document which projects opt-in to managing their own CoC reporting.

That's already covered and not subject to disagreement.

Reply:
Fixed in 08811e8. @bnb mind looking at the language and seeing what you think of it? Thanks.